aea/cli/run.py

```diff
@@ -282,7 +283,7 @@ def run(
 
     aea = _build_aea(ctx, connection_ids)
 
-    click.echo(AEA_LOGO)
+    click.echo(AEA_LOGO + "v" + __version__ + "\n")
```

Comment:
we add the version below the logo to make it more informative

Reply:
probably better keep the logo only in `aea init` and `aea version` (which we don't suport atm, only `aea --version`).

O/w it is going to be annoying.

Reply:
hmm so revert it?

Reply:
In `aea run` I'd say yes.